WordPress/gutenberg
# Remove experimental setting for interactivity API and behaviors

Unblocks #51156

## What?

Remove Behaviors and Interactivity API from Gutenberg experiments. Behaviors are still disabled by default, and need to be enabled using `theme.json`, Global Styles or block settings.

## Why?

Behaviors are ready to be widely tested. We have two weeks for the next Gutenberg release and we can also, after some more testing, include the backports for 6.4

## How?

Removing some conditionals.

## Testing Instructions




Enable behaviors with the three options available. Check that image lightbox is working as expected.

### Testing Instructions for Keyboard


## Screenshots or screencast 
Before:
![Screenshot 2023-07-21 at 13 54 55](https://github.com/WordPress/gutenberg/assets/37012961/5208083e-ad4f-4afe-91a8-8b31ff0f852b)


After:
![Screenshot 2023-07-21 at 13 52 44](https://github.com/WordPress/gutenberg/assets/37012961/554d2bff-9980-41d4-ab28-6824b652a555)
